# Array of fused buoys (LiDAR tracks with color from CV)

std_msgs/Header header
FusedBuoy[] buoys
